README.md

```diff
@@ -1,97 +1,105 @@
-This is a PostgreSQL Docker image to automatically upgrade
-your database.
+# pgautoupgrade
 
-Its whole purpose in life is to automatically detect the
-version of PostgreSQL used in the existing PostgreSQL data
-directory, then automatically upgrade it (if needed) to the
-required version of PostgreSQL.
+This is a PostgreSQL Docker image to automatically upgrade your database.
 
-After this, the PostgreSQL server starts and runs as per
-normal.
+Its whole purpose in life is to automatically detect the version of PostgreSQL used in the existing PostgreSQL data directory, then automatically upgrade it (if needed) to the required version of PostgreSQL using `pg_upgrade` with the `--link` option.
 
-The reason this Docker image is needed, is because the
-official Docker PostgreSQL image has no ability to handle
-version upgrades, which leaves people to figure it out
-manually (not great): https://github.com/docker-library/postgres/issues/37
+After this, the PostgreSQL server starts and runs as per normal. The old cluster data will be removed.
```

Comment:
Actually, I see that `~/delete_old_cluster.sh` is removed – but I am not sure where it gets executed. Some clarification would be good on this :)